modules/users/client/config/users.client.routes.js

```diff
@@ -101,20 +101,6 @@
         url: '/reset',
         template: '<ui-view/>'
       })
-      .state('password.reset.invalid', {
```

Comment:
I'd prefer to keep the routes/views. We could always open up a discussion on how to change this flow, but I think having that discussion would distract us from the purpose of this PR.

I'm open to the discussion though, if others feel this should be addressed now.